test/v1alpha1/dag_test.go

```diff
@@ -34,6 +35,8 @@ import (
 	knativetest "knative.dev/pkg/test"
 )
 
+const sleepDuration = 30 * time.Second
```

Comment:
30s feels like a long time to wait in a test?

Reply:
Waiting for a rerun of the e2e tests is a lot longer 😉 

I'm happy to lower this, but not sure what you are comfortable with.  5s is the 99P scheduling latency on K8s when there's available capacity, and running these tests with `t.Parallel()` on KinD things get busy quickly.  I've seen 9s apart in recently memory, but think I've seen up to 13s.

Reply:
(to be clear the `9s` and `13s` are *failures* with what's at HEAD)

Reply:
I reduced this to 15s, which is larger than I can recall seeing this fail with.